### ISSUE TITLE
Questions about data division

### ISSUE THREAD

**Issue by Friday-night7** [dyh127/RFNet#4]
Dear author,
           After reading your paper, I admire your work very much. The paper says “ BRATS2020 contains 369 training subjects which are
randomly split by us into 219, 50 and 100 subjects for training, validation and test, respectively  ” . I would like to know exactly how you divide the training set, validation set and test set, is it by case order or random? 

          "if args.dataname in ['BRATS2020', 'BRATS2015']:
                  train_file = 'train.txt' "

           In the above code, I want to know whether your training data index stored in the "train.txt"?  But I couldn't find this file, so I would like you to tell me how to divide the data set.
           Looking forward to your reply. Thanks.

**Comment by dyh127** [comment]
The data is split randomly and you can find the split files in the downloaded data folders (BRATS2020_Training_none_npy.tar.gz ...) .
